inputremapper/gui/components/editor.py

```diff
@@ -294,11 +199,14 @@ def __init__(
         # set the name or combination.beautify as label
         self.label.set_label(self.name)
 
+        self.label.set_margin_top(11)
+        self.label.set_margin_bottom(11)
+
         # button to edit the name of the mapping
         self.edit_btn = Gtk.Button()
         self.edit_btn.set_relief(Gtk.ReliefStyle.NONE)
         self.edit_btn.set_image(
-            Gtk.Image.new_from_icon_name(Gtk.STOCK_EDIT, Gtk.IconSize.MENU)
+            Gtk.Image.new_from_icon_name("text-x-generic", Gtk.IconSize.MENU)
```

Comment:
I feel like the edit symbol is better at communicating the purpose of the button.

![image](https://user-images.githubusercontent.com/31804124/196028053-e70d0c46-7281-43fe-84f8-9ee11935ba08.png)
![image](https://user-images.githubusercontent.com/31804124/196028138-6bddf43a-370f-4877-b12c-d07bb5e91355.png)

Reply:
Stock icons seem to be deprecated (at least glade claims that it is). The `Gtk.STOCK_EDIT` icon isn't integrating well with other icons in GNOME.

But you are right, the STOCK_EDIT is still better

Reply:
```python
        self.edit_btn.set_image(
            Gtk.Image.new_from_icon_name("edit", Gtk.IconSize.MENU)
        )
```
seems to work just fine.

Reply:
not in gnome: ![image](https://user-images.githubusercontent.com/28510156/196035076-78b45fcc-d5ec-4983-8e69-37ecf50a8d0d.png)